src/components/NetworkSelector/NetworkSelector.js

```diff
@@ -75,7 +80,7 @@ function NetworkSelector(props) {
           {options.v3.map((network) => (
             <DropdownItem
               key={`v3-${network.value}`}
-              active={currentVersion === "v3" && currentNetwork.value === network.value}
+              //        active={currentVersion === "v3" && currentNetwork.value === network.value}
```

Comment:
why is this changed?

Reply:
same, as above it crashes if arbitrum is not in network selector

Reply:
You can try to validate with a shorthand if statement, maybe something like this;

`{(currentVersion === "v3" && currentNetwork.value === network.value) ? network.value != arbitrum : null}`